code/datums/uplink_item.dm

```diff
@@ -100,7 +100,7 @@ var/list/uplink_items = list()
 	category = "Conspicuous and Dangerous Weapons"
 
 /datum/uplink_item/dangerous/pistol
-	name = "Syndicate Pistol"
+	name = "Stetchkin Pistol"

```

Comment:
bear, this was the pistols name before Paprika changed it into a generic name. So this is actually just me changing it back.